bude_hezky/request/request_maker.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from requests.exceptions import HTTPError
+
+SUCCESSFUL_STATUS_CODE_THRESHOLD = 299;
```

Comment:
středník

![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia.giphy.com%2Fmedia%2F12sgzt0oxFUkoM%2Fgiphy.gif&f=1&nofb=1)

Reply:
Budeš to mít se mnou těžký, než vybiješ tyhle zlozvyky :D